src/win/core.c

```diff
@@ -80,6 +81,100 @@ static void uv__crt_invalid_parameter_handler(const wchar_t* expression,
 }
 #endif
 
+static uv_loop_t** uv__loops;
+static int uv__loops_size;
+static int uv__loops_capacity;
+#define UV__LOOPS_CHUNK_SIZE 8
+static uv_mutex_t uv__loops_lock;
+
+static void uv__loops_init() {
+  int i;
+  uv_mutex_init(&uv__loops_lock);
+  uv__loops = uv__malloc(sizeof(uv_loop_t*) * UV__LOOPS_CHUNK_SIZE);
```

Comment:
NULL does not have to be all bits zero, but it does not really matter here. I'll change that.